server/tables/tables.go

```diff
@@ -545,6 +546,7 @@ func PostAutoMigrate(db *gorm.DB) error {
 		"invocations_stats_user_index":        "(`group_id`, `user`, `action_count`, `duration_usec`, `updated_at_usec`, `success`, `invocation_status`)",
 		"invocations_stats_host_index":        "(`group_id`, `host`, `action_count`, `duration_usec`, `updated_at_usec`, `success`, `invocation_status`)",
 		"invocations_stats_repo_index":        "(`group_id`, `repo_url`, `action_count`, `duration_usec`, `updated_at_usec`, `success`, `invocation_status`)",
+		"invocations_stats_branch_index":      "(`group_id`, `branch_name`, `action_count`, `duration_usec`, `updated_at_usec`, `success`, `invocation_status`)",
```

Comment:
This looks good but please test and make sure all the stats and trends pages are still hitting these indexes

Reply:
Tested it; trends certainly is. What stats pages are you talking about?

Reply:
I meant like when you hit https://app.buildbuddy.io/#repos or https://app.buildbuddy.io/#hosts or whatever -- are those still hitting an index.

Reply:
Checked #repos; looks like it is.